### PR TITLE
a little addition to the welcome email for data requests

### PR DESCRIPTION
**Pull request by Baalmart**
# a little addition to the welcome email for data requests

**_WHAT DOES THIS PR DO?_**
Just a little addition to the welcome email for data requests

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
N/A

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/airqo-platform/AirQo-api/tree/join-req-message

**_HOW DO I TEST OUT THIS PR?_**
```
cd airqo-api/src/auth-service
npm insall
npm run dev-mac
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] Request data access:
http://localhost:3000/api/v1/users/candidates/register?tenant=airqo

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A

**_ARE THERE ANY RELATED PRs?_**
N/A

### CODE REVIEW

File: src/auth-service/utils/email.msgs.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ module.exports = {
       `Dear ${firstName + " " + lastName} \n\n` +
       "Your request to join the AirQo platform has been received.\n" +
       "We shall get back to you as soon as possible. \n\n" +
+      "Before utilising airqo data, your application record has to undergo \n" +
+      "the process of approval by platform administration. Once your \n" +
+      "application reviewed, you will receive an email on the mailbox \n\n" +
```
Comment (danielogen) [inline src/auth-service/utils/email.msgs.js:24]: we might be missing a word here! ...once your application *is* reviewed...

**Review by danielogen — CHANGES_REQUESTED** [review]
LGTM -  just a tiny change needed

File: src/auth-service/utils/email.msgs.js [hunk 2]
```diff
@@ -19,6 +19,9 @@ module.exports = {
       `Dear ${firstName + " " + lastName} \n\n` +
```
Comment (ja-odur) [inline src/auth-service/utils/email.msgs.js:19]: Not sure but dont we need a comma (,) after lastName @Baalmart?

**Review by ja-odur — CHANGES_REQUESTED** [review]
Test and works fine. Just a few typos to fix

File: src/auth-service/utils/email.msgs.js (hunk 2 above)
Comment (Baalmart) [reply src/auth-service/utils/email.msgs.js]: Resolved. @ja-odur

File: src/auth-service/utils/email.msgs.js (hunk 1 above)
Comment (Baalmart) [reply src/auth-service/utils/email.msgs.js]: Resolved @danielogen

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**